frontend/public/style/_overrides.scss

```diff
@@ -497,3 +497,8 @@ button.pf-c-dropdown__menu-item.pf-m-disabled {
 #modal-container .pf-c-backdrop {
   position: absolute !important;
 }
+
+.text-muted{
+  color: var(--pf-global--Color--200);
+  font-size: var(--pf-global--FontSize--xs);
```

Comment:
We don't want to change the font-size all of muted text. We should this style on a lot of text in the console that is different sizes.